### PR TITLE
Add PHPCS config and fix errors

### PR DESCRIPTION
**Pull request by szepeviktor**
Now we pass

```bash
vendor/bin/phpcs . --standard=WordPress-Core --exclude=WordPress.PHP.StrictComparisons,WordPress.WP.I18n
```

Restart of #31

### CODE REVIEW

**Comment by szepeviktor** [comment]
> WordPress.PHP.StrictComparisons

I do not dare to touch those as I don't know what actual values are compared.

> WordPress.WP.I18n

I'm not up to phrasing instructions for translators.

File: phpunit/tests/term-meta.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName
```
Comment (ocean90) [inline phpunit/tests/term-meta.php:1]: This should be added as a exclude rule to the `phpunit.xml.dist` file.

File: src/class-wp-import.php [hunk 2]
```diff
@@ -229,25 +231,25 @@ function get_authors_from_import( $import_data ) {
...
 		<li><?php $this->author_select( $j++, $author ); ?></li>
 <?php endforeach; ?>
 	</ol>
 <?php endif; ?>
 
-<?php if ( $this->allow_fetch_attachments() ) : ?>
+		<?php if ( $this->allow_fetch_attachments() ) : ?>
 	<h3><?php _e( 'Import Attachments', 'wordpress-importer' ); ?></h3>
 	<p>
 		<input type="checkbox" value="1" name="fetch_attachments" id="import-attachments" />
```
Comment (ocean90) [inline src/class-wp-import.php:255]: This doesn't look correct, see https://github.com/szepeviktor/wordpress-importer/blob/phpcs/src/class-wp-import.php#L228-L263.

File: phpunit/tests/term-meta.php (hunk 1 above)
Comment (szepeviktor) [reply phpunit/tests/term-meta.php:1]: PHPCS does not provide a way to exclude rules for certain files only :(

What to do now?

File: src/class-wp-import.php (hunk 2 above)
Comment (szepeviktor) [reply src/class-wp-import.php]: Done.

File: phpunit/tests/term-meta.php (hunk 1 above)
Comment (jrfnl) [reply phpunit/tests/term-meta.php:1]: It most definitely does.

```xml
<rule ref="WordPress.Files.FileName">
    <exclude-pattern>*/phpunit/tests/*\.php$</exclude-pattern>
</rule>
```

Note: exclude patterns are treated like PCRE regexes with one caveat, that all `*`s are automatically converted to `.*`.

Comment (szepeviktor) [reply phpunit/tests/term-meta.php]: Thank you @jrfnl!

Done.

File: .travis.yml [hunk 3]
```diff
@@ -50,6 +50,11 @@ matrix:
 before_script:
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
   - phpenv config-rm xdebug.ini || echo 'No xdebug config.'
+  - |
+    if [[ "$TRAVIS_PHP_VERSION" == "7.3" ]]; then
+      composer global require wp-coding-standards/wpcs phpcompatibility/phpcompatibility-wp dealerdirect/phpcodesniffer-composer-installer
+      phpcs
+    fi
```
Comment (ocean90) [inline .travis.yml:57]: `before_script` shouldn't run any scripts.  It's better to have a separate job for phpcs.

Comment (szepeviktor) [reply .travis.yml]: Done.

**Comment by szepeviktor** [comment]
Thank you.

**Pull request merged**